OxfordBodleian/Juel-Jensen/BDLaethe35.xml

```diff
@@ -227,6 +751,157 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+                                        <del rend="encircled">ዘን</del> ዘገብረ፡ <sic>እዚአብሔር፡</sic> 
+                                        ፀፃዳሊ<supplied reason="omitted">፡</supplied> <sic>በኖበ አቡ</sic> አለቃ<supplied reason="omitted">፡</supplied> 
+                                        ገ<supplied reason="omitted">ብ</supplied>ረ<supplied reason="omitted">፡</supplied> እ
+                                    </q>
+                                </item>
+                                
+                                <item xml:id="e1">
+                                    <desc>
+                                        Letters and word have been erased (e.g. <locus target="#43v #71r #148rb"/>),
+                                        sometimes with angular symbols filling the lines,
+                                        and letters have been marked for deletion by over- and underlining
+                                        or encircling
+                                        (e.g. <locus target="#9r #87r #96r"/>),
+                                        or blotted out (e.g. <locus target="#14r #64v #81v"/>).
+                                        
+                                        Letters and words have been added interlineally (e.g. <locus target="#21v #36r #63v"/>),
+                                        over erasures (e.g. <locus target="#13v #76r #150rb"/>),
+                                        and, at least once, in the upper margin with a <hi rendition="simple:italic">tamallas</hi>-sign
+                                        indicating their position in the text (e.g. <locus target="# #57v #"/>).
+                                        
+                                        Compressed script (e.g. <locus target="#4r #104v #116v"/>), line take-ups
+                                        (e.g. <locus target="#4r #51r #91v"/>), and, more rarely, line take-downs (e.g. <locus target="#24v #53r"/>)
+                                        are used.</desc>
+                                </item>
+                                
+                                <item xml:id="e2">
+                                    <desc>A white sticker is glued to the inner side of the front board, in the middle. The name 
+                                        of <persName ref="PRS5782JuelJen"/> is printed on it
+                                        (‘<foreign xml:lang="gez">ቤንት፡ ዩውል፡ የንሰን።</foreign>’) and the signatures ‘MS Aeth. e. 35’ is written in pencil.
+                                        An additional sticker, with the signature ‘Ms 23’ on it, is glued below the previous one.</desc>
+                                </item>
+                                
+                                <item xml:id="e3">
+                                    <locus target="#59r"/>
+                                    <desc>Secondary note marking the mid-point of <ref target="#ms_i1.1"/>.</desc>
```

Comment:
"the mid-point of `<ref target="#ms_i1.1">the Psalms of David</ref>`."

Btw, I think I have used midpoint (not mid-point) in my descriptions.